modules/RTC/JitsiLocalTrack.js

```diff
@@ -338,7 +338,6 @@ export default class JitsiLocalTrack extends JitsiTrack {
         if (this._streamEffect) {
             this._streamEffect.stopEffect();
             this._setStream(this._originalStream);
-            this._originalStream = undefined;
```

Comment:
what is this change about?

Reply:
I will have to keep the original stream around so that I can remove it from the _addedStreams array in TraceablePeerconnection after the media stream is replaced.